.github/dependabot.yml

```diff
@@ -1,16 +1,20 @@
 version: 2
+
 updates:
-- package-ecosystem: bundler
-  directory: "/"
-  schedule:
-    interval: daily
-    time: "02:00"
```

Comment:
Why did we remove the 2am time?

Reply:
Thanks @jemmaissroff ! I wanted to remove unnecessarily specific configuration (in the spirit of _convention over configuration_).

In this case, note that if you don't provide the exact time, it [defaults to 5am UTC](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#scheduleinterval), which is currently 1am in eastern time. Do you see benefit in preferring a more specific configuration?

Reply:
Makes sense to me! Wasn't defending either side, just wanted to make sure it was intentional and wasn't a necessary configuration detail! And thanks for checking the default